# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lesstif/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Well - they already commented "Only include this when necessary. What is this
really dependent on ?". I think the answer is LESSTIF_PRODUCTION. Since
we enable "production" in configure we need to not include this function
since the ...DebugFile is not defined ...

  - Rene Rebe <rene@exactcode.de>

--- lesstif-0.93.96/lib/Xm-2.1/DebugUtil.c.orig	2004-09-29 11:17:26.904106539 +0000
+++ lesstif-0.93.96/lib/Xm-2.1/DebugUtil.c	2004-09-29 11:17:51.370195643 +0000
@@ -2517,6 +2517,7 @@
  * Only include this when necessary.
  * What is this really dependent on ?
  */
+#ifndef LESSTIF_PRODUCTION
 extern void
 _LtDebugPrintStackTrace()
 {
@@ -2534,3 +2535,4 @@
 	fflush(_LtDebugFile);
 #endif
 }
+#endif
